.circleci/config.yml

```diff
@@ -159,16 +159,6 @@ workflows:
     jobs:
       - check-code-formatting
       - build-and-test
-      - assume-role-development:
```

Comment:
Why are we removing these jobs from the pipeline?

Reply:
I thought we are supposed to only deploy from master these days, not from branches? which is the right config.yml to use?

Reply:
Yes that's right, we only want to deploy when we are on the master branch, so we need to add filters to the pipeline rather than removing the whole job (if you remove it, it would never deploy to dev)
See an example [here](https://github.com/LBHackney-IT/person-api/blob/master/.circleci/config.yml#L167-L185) of adding the filters.

Reply:
All done.